payroll/models/hr_payroll_structure.py

```diff
@@ -20,7 +20,7 @@ def _get_parent(self):
         return self.env.ref("hr_payroll.structure_base", False)
 
     name = fields.Char(required=True)
-    code = fields.Char(string="Reference", required=True)
+    code = fields.Char(string="Reference", required=False)
```

Comment:
We should remove the required key instead of setting it to false. Because default value is false if not informed.

Reply:
Done